### PR TITLE
K8s executor 2

### PR DESCRIPTION
**Pull request by dimberman**
Merging all of the work we've done on k8s-executor-2 back into airflow-kubernetes-executor. @grantnicholas @benjigoldberg please review.

### CODE REVIEW

File: airflow/bin/cli/cli_factory.py [hunk 1]
```diff
@@ -1,501 +0,0 @@
-from collections import namedtuple
```
Comment (grantnicholas) [inline airflow/bin/cli/cli_factory.py]: Are these changes from upstream airflow? just not sure why we would need to modify this file.

**Comment by grantnicholas** [comment]
outside of the cli stuff, looks like it's good.

**Comment by benjigoldberg** [comment]
Agree w/ @grantnicholas lgtm as well

File: airflow/bin/cli/cli_factory.py (hunk 1 above)
Comment (dimberman) [reply airflow/bin/cli/cli_factory.py]: @grantnicholas This is actually bringing the cli stuff back to how master is. I had made some refactors on the airflow-kubernetes-executor branch, but realize those should be submitted seperately.

**Pull request merged**